src/core/meshcop/border_agent.cpp

```diff
@@ -252,6 +252,7 @@ BorderAgent::BorderAgent(Instance &aInstance)
     , mUdpProxyPort(0)
     , mUdpReceiver(BorderAgent::HandleUdpReceive, this)
     , mTimer(aInstance)
+    , mPskcChangedDuringConnectedOrAccepted(false)
```

Comment:
Can we maybe use a shorter name for this?
- How about `mPskcChanged`?
- Or maybe we can use a name to indicate action we want to be performed, `mRestartOnDisconnect` i.e., restart agent after current connection is disconnected?

Thoughts?

Reply:
changed to mPskcChanged in new patch, pls help to take another review.